demo/full/scripts/modules/player/events.js

```diff
@@ -105,6 +105,7 @@ const linkPlayerEventsToState = (player, state, $destroy) => {
       stateUpdates.availableVideoTracks = [];
       stateUpdates.availableLanguages = [];
       stateUpdates.availableSubtitles = [];
+      stateUpdates.lowLatencyMode = false;
```

Comment:
We might want to also reset the playback rate now if we have no other good solution.